development/components/form/types-reference/translatable.md

```diff
@@ -15,7 +15,7 @@ The `TranslatableType` allows you to configure multilanguage input. This multila
 | locales | array  | Enabled shop locales (languages)                      | Locales in which field can be translated |
 
 {{% notice info %}}
-If you wish to use [FormattedTextareaType]({{< ref "1.7/development/components/form/types-reference/formatted-textarea" >}}) as type, your base type must be `TranslateType` instead of `TranslatableType`.
+If you wish to use [FormattedTextareaType]({{< ref "1.7/development/components/form/types-reference/formatted-textarea" >}}) as type, your base type must be `TranslateType` instead of `TranslatableType`. Do not forget to add the option `hideTabs` at **true** if you want th display the languages list above the WYSIWYG.
```

Comment:
```suggestion
If you wish to use [FormattedTextareaType]({{< ref "1.7/development/components/form/types-reference/formatted-textarea" >}}) as type, your base type must be `TranslateType` instead of `TranslatableType`. Do not forget to add the option `hideTabs` at **true** if you want to display the languages list above the WYSIWYG.
```